components/helper/ShareButtons.tsx

```diff
@@ -1,4 +1,7 @@
-import { Grid } from '@mui/material';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+import EmailIcon from '@mui/icons-material/Email';
+import { Box, Grid, Typography } from '@mui/material';
+import PropTypes from 'prop-types';
```

Comment:
We shouldn't need PropTypes here since we are using typescript and declaring a type.